docs/components/alerts/index.html

```diff
@@ -1,161 +1,114 @@
-        </div>
+    <h2 id="default">Demos</h2>
+    <div id="vue-alertDemo" class="hxRow" v-cloak>
+      <div class="hxCol hxSpan-12-xs hxSpan-3-lg hxOrder-2-lg">
+        <h3>Options</h3>
+        <p>
+          <label>
+            <input type="checkbox" v-model="isClosable">
+            Closable
+          </label>
+        </p>
+        <p>
+          <b>Variant:</b><br />
+          <select v-model="variant">
+            <option v-for="_variant in variants" :value="_variant">
+              {% raw %}{{ _variant.label }}{% endraw %}
+            </option>
+          </select>
+        </p>
       </div>
-    </div>
-    {% code 'html' %}
-      <div role="status" class="hxAlerts" aria-live="polite">
-        <div class="hxAlert">
-          <div class="hxAlert__icon">
-            <hx-icon type="info-circle"></hx-icon>
-          </div>
-          <div class="hxAlert__text">
-            <span class="hxAlert__status">Info</span>
-            Here is helpful information.
-          </div>
-          <div class="hxAlert__action">
-            <a href="#">Acknowledge</a>
-          </div>
-          <button class="hxAlert__dismiss"><hx-icon type="times"></hx-icon></button>
-        </div>
-      </div>
-    {% endcode %}
-  </section>
-
-  <section>
-    <h2 id="error">Error</h2>
 
-    <div class="demo">
-      <div role="status" class="hxAlerts" aria-live="polite">
-        <div class="hxAlert hxAlert--error">
-          <div class="hxAlert__icon">
-            <hx-icon type="exclamation-circle"></hx-icon>
-          </div>
-          <div class="hxAlert__text">
-            <span class="hxAlert__status">Error</span>
-            Change a few things up and try submitting again.
-          </div>
-          <div class="hxAlert__action">
-            <a href="#">Correct Issue</a>
-          </div>
-          <button class="hxAlert__dismiss"><hx-icon type="times"></hx-icon></button>
-        </div>
-      </div>
-    </div>
-    {% code 'html' %}
-      <div role="status" class="hxAlerts" aria-live="polite">
-        <div class="hxAlert hxAlert--error">
-          <div class="hxAlert__icon">
-            <hx-icon type="exclamation-circle"></hx-icon>
-          </div>
-          <div class="hxAlert__text">
-            <span class="hxAlert__status">Error</span>
-            Change a few things up and try submitting again.
-          </div>
-          <div class="hxAlert__action">
-            <a href="#">Correct Issue</a>
-          </div>
-          <button class="hxAlert__dismiss"><hx-icon type="times"></hx-icon></button>
+      <div class="hxCol hxSpan-12-xs hxSpan-9-lg hxOrder-1-lg">
+        <h3>Basic Alert</h3>
+        <div class="demo alert-demo">
+          <hx-disclosure :disabled="!isClosable" class="hxBtn" aria-controls="myAlert">
+            Toggle Alert
+          </hx-disclosure>
+          <br /><br />
+          <hx-alert id="myAlert" :closable="isClosable" v-bind:class="cssClasses">
```

Comment:
More often than not, an alert should be closable by default. Instead of using a `closable` attribute to _add_ the ability to close, what about inverting the logic with a `static` attribute to _remove_ the ability to close?

_current_
```html
<hx-alert closable><!-- this alert can be closed --></hx-alert>
<hx-alert><!-- this alert cannot be closed --></hx-alert>
```

_proposed_
```html
<hx-alert><!-- this alert can be closed --></hx-alert>
<hx-alert static><!-- this alert cannot be closed --></hx-alert>
```

Reply:
I'm not married to the name of the `static` attribute, either. I'm open to suggestions.

Reply:
:+1: